### ISSUE TITLE
Release 2021.5.2 is broken

### ISSUE THREAD

**Issue by yurtaev** [cloudflare/homebrew-cloudflare#27]
```
==> Upgrading cloudflare/cloudflare/cloudflared 2021.5.1 -> 2021.5.2
==> Downloading https://packages.argotunnel.com/dl/cloudflared-2021.5.2-darwin-amd64.tgz
Already downloaded: /Users/yurtaev/Library/Caches/Homebrew/downloads/83ffb733858e203c39ba17387671f83c7791b76e96134eadef5829ce388a519b--cloudflared-2021.5.2-darwin-amd64.tgz
Error: An exception occurred within a child process:
  RuntimeError: Empty archive
```

`cloudflared-2021.5.2-darwin-amd64.tgz` is only 45 bytes

```
➜  ~ ls -lh Downloads/cloudflared-2021.5.2-darwin-amd64.tgz
-rw-rw-rw-@ 1 yurtaev  staff    45B May 14 10:15 Downloads/cloudflared-2021.5.2-darwin-amd64.tgz

➜  ~ sha256sum -t Downloads/cloudflared-2021.5.2-darwin-amd64.tgz
65a46df360fdd93722be999654eacabd2087ed368b136bcef58c3642e91e34a2  Downloads/cloudflared-2021.5.2-darwin-amd64.tgz
```

**Comment by jared-wilkinson** [comment]
Same issue on 2021.5.3:

```
➜  ~ ll cloudflared-2021.5.3-darwin-amd64.tgz                                                                                                                                                          <aws:prototype>
-rw-r--r--  1 jawilkinson  staff    45B May 14 11:17 cloudflared-2021.5.3-darwin-amd64.tgz
```

Results in `Error: An exception occurred within a child process: RuntimeError: Empty archive` during `brew install`.

**Comment by flavioheleno** [comment]
Same applies to 2021.5.4
```bash
$ ll cloudflared-2021.5.4-darwin-amd64.tgz
-rw-r--r--  1 flavio  staff    45B May 14 16:46 cloudflared-2021.5.4-darwin-amd64.tgz
```

**Comment by yurtaev** [comment]
fixed 2021.5.6
